environments/eck-ror/start.sh

```diff
@@ -90,12 +90,38 @@ if [[ -z $ES_VERSION || -z $KBN_VERSION ]]; then
   show_help
 fi
 
+DOCKERFILE_DIR="./images/ubuntu"
+IMAGE_NAME="node-apm-app"
+TAG="latest"
+
+docker build -t "$IMAGE_NAME:$TAG" "$DOCKERFILE_DIR"
+
+if [ $? -eq 0 ]; then
```

Comment:
there is `set -e` at the beginning of the script. When the previous command fails, the script won't reach this point.